cirrus/google_cloud/manager.py

```diff
@@ -134,64 +134,41 @@ def __exit__(self, exception_type, exception_value, traceback):
 
     def create_proxy_group_for_user(self, user_id, username):
         """
-        Creates a proxy group for the given user, creates a service account
-        for the user, and adds the service account to the group.
+        Creates a proxy group for the given user, does NOT create a 
+        service account for the proxy group
```

Comment:
I don't think you need to say it does not create a service account, just take that part out completely